scripts/run_integration_tests.sh

```diff
@@ -78,24 +78,40 @@ popd
 
 # Build custom kernel based on virtio-pmem and virtio-fs upstream patches
 VMLINUX_IMAGE="$WORKLOADS_DIR/vmlinux"
+VMLINUX_PVH_IMAGE="$WORKLOADS_DIR/vmlinux.pvh"
 BZIMAGE_IMAGE="$WORKLOADS_DIR/bzImage"
 
-LINUX_CUSTOM_DIR="linux-custom"
+LINUX_CUSTOM_DIR="$WORKLOADS_DIR/linux-custom"
 
-if [ ! -f "$VMLINUX_IMAGE" ]; then
+if [ ! -f "$VMLINUX_IMAGE" -o ! -f "$VMLINUX_PVH_IMAGE" ]; then
```

Comment:
Can you use `||` instead of `-o`?

Reply:
Sure. Will change it to this in the next revision:

```bash
if [ ! -f "$VMLINUX_IMAGE" ] || [ ! -f "$VMLINUX_PVH_IMAGE" ]; then
...
```